### ISSUE TITLE
'r?' without auto-assignment

### ISSUE THREAD

**Issue by RalfJung** [rust-lang/triagebot#1674]
In Miri I'd like to use `r?` but we don't currently want automatic assignment of PR reviewers. However, according to the [docs](https://github.com/rust-lang/triagebot/wiki/Assignment#pr-assignment):
> If the owners map is configured, then triagebot will automatically select a reviewer based on which files were modified in the PR. This also enables the ability for users to post a comment with r? name to set the assignment to a specific user.

Is there a way to enable `r?` comments without having an owners map?

**Comment by ehuss** [comment]
Not really. You can use `@rustbot assign @user` if you are an team member.

**Comment by RalfJung** [comment]
I guess that works, though `r?` is so much shorter, would be nice to also be able to use that.

**Comment by apiraino** [comment]
Don't we use `r? @ghost` for this purpose?

I'd close this issue because I *think* it's solved, but feel free to reopen if I misunderstood :)

**Comment by Kobzol** [comment]
I think the idea was to allow using r? without auto assignment, but also *without* having to write `r? @ghost` on every PR. I think this is possible today, pretty sure at least rustc-dev-guide is configured in such a way (maybe also miri).

**Comment by apiraino** [comment]
> Then we'd have to ensure every contributor puts that in the PR description.^^

mhmm yeah ~~fail~~ fair point ^^

> But yeah, this works now. Only the docs are still a bit confusing since most of [forge.rust-lang.org/triagebot/pr-assignment.html](https://forge.rust-lang.org/triagebot/pr-assignment.html) clearly assumes that you use automatic assignment.

In that page there a small paragraph about using ghost. Would it help rewording/remixing the two paragraphs about assignment?
